frontend/.eslintrc.js

```diff
@@ -34,8 +34,6 @@ module.exports = {
     // This will display prettier errors as ESLint errors.
     // Make sure this is always the last configuration in the extends array.
     "plugin:prettier/recommended",
-    // Recommended Jest configuration to enforce good testing practices
```

Comment:
is there some vi equiv we should be extending?

Reply:
Good call. Will replace with vi equivalent and disable any eslint rules that violate it per our new setup.